grabbit.sh

```diff
@@ -30,7 +30,7 @@ function newGrabbitRequest() {
     echo $SET_YELLOW
     echo "Processing....."
     echo $SET_NO_COLOR
-    curl -s -f -X PUT --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
+    curl -s -f -X PUT --header "Content-Type: text/plain" --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
```

Comment:
This seems necessary to correctly `PUT` the config file to AEM 6.2 correctly. Without this, in AEM 6.2... the content of the file gets wrongly placed into weirdly managed Request Parameters instead.

This continues to work as it is on AEM 6.1.